packages/hash/frontend/src/components/hooks/blockProtocolFunctions/knowledge/useBlockProtocolGetEntity.ts

```diff
@@ -17,10 +17,7 @@ export const useBlockProtocolGetEntity = (): {
   const [getEntityTypeRootedSubgraphFn] = useLazyQuery<
     GetEntityTypeRootedSubgraphQuery,
     GetEntityTypeRootedSubgraphQueryVariables
-  >(getEntityTypeRootedSubgraphQuery, {
-    /** @todo reconsider caching. This is done for testing/demo purposes. */
-    fetchPolicy: "no-cache",
-  });
+  >(getEntityTypeRootedSubgraphQuery);
```

Comment:
What is the default behaviour here now? Is it still `no-cache`?

Reply:
Ah good point, perhaps we should be making use of the hook here instead? 
The default would be `cache-first` which is what we would want for any `get*Type` query as the IDs refer to immutable records

Reply:
I think I do want to replace this with the hook, actually. It is temporary, but the code with the hook instead is way shorter